File: ModelDataProducer/src/main/java/com/modelDataProducer/PizzaStore/model/Order.java
package com.modelDataProducer.PizzaStore.model;

import java.util.List;

public class Order {
    private CreditCard creditCardInfo;
    private List<OrderedMenuItem> orderedMenuItems;

    /**
     * @return CreditCard return the creditCardInfo
     */
    public CreditCard getCreditCardInfo() {
        return creditCardInfo;
    }

    /**
     * @param creditCardInfo the creditCardInfo to set
     */
    public void setCreditCardInfo(CreditCard creditCardInfo) {
        this.creditCardInfo = creditCardInfo;
    }

    /**
     * @return List<OrderedMenuItem> return the orderedMenuItems
     */
    public List<OrderedMenuItem> getOrderedMenuItems() {
        return orderedMenuItems;
    }

    /**
     * @param orderedMenuItems the orderedMenuItems to set
     */
    public void setOrderedMenuItems(List<OrderedMenuItem> orderedMenuItems) {
        this.orderedMenuItems = orderedMenuItems;
    }
}

File: Mediator/Models/EntityModels/Order.cs
using System;
using System.Collections.Generic;

namespace PizzaStore.Model.Entity
{
    public class Order
    {
        public string Id { get; set; } = Guid.NewGuid().ToString();
        public CreditCard CreditCardInfo { get; set; }
        public List<OrderedMenuItem> OrderedMenuItems { get; set; }
    }
}